### ISSUE TITLE
If a CDN domain isn't findable in the repo, is it still worth contributing?

### ISSUE THREAD

**Issue by kurtextrem** [patrickhulce/third-party-web#237]
Hi,

At PerfNow, Jack Franklin from the Chrome DevTools team mentioned, they source 3rd-party names from this repo, to e.g. show "jQuery CDN" instead of the full URL in the Chrome DevTools Performance tab's 3rd-party "insight" banner.
This now brings me to the question: I work for Framer, which is a website builder platform. We host the JavaScript for websites our customers/users create on `https://framerusercontent.com` - so it'd be nice to name it "Framer CDN" inside DevTools. 
However, it seems like our domain isn't in the dataset, which probably means not enough origins make use of it (yet). Does this mean I should rather not submit a PR to update `data/entities.js` for now, or is this something you'd like to see?

**Comment by patrickhulce** [comment]
> Does this mean I should rather not submit a PR to update data/entities.js for now, or is this something you'd like to see?

Yes we'd love a PR from you to update to the correct URLs for your entity :)

**Comment by kurtextrem** [comment]
Thanks! Done: https://github.com/patrickhulce/third-party-web/pull/245
